website/redirects.js

```diff
@@ -1,6 +1,25 @@
 // Add all server-side redirects here. Will be loaded by next.config.mjs.
 
 module.exports = [
+  /*
+   *
+   * Android Client
+   *
+   */
+  {
+    source: "/dl/firezone-client-android/latest/apk",
+    destination:
+      // mark:current-android-version
+      "https://www.github.com/firezone/firezone/releases/download/android-client-1.4.0/firezone-android-client-1.4.0.apk",
+    permanent: false,
+  },
+  {
+    source: "/dl/firezone-client-android/latest/aab",
+    destination:
+      // mark:current-android-version
+      "https://www.github.com/firezone/firezone/releases/download/android-client-1.4.0/firezone-android-client-1.4.0.aab",
```

Comment:
is there any value in offering an AAB for download? If I understand https://developer.android.com/guide/app-bundle correctly, it is mainly for Google Play?

Reply:
Yeah probably a correct assumption. Updated.